webrtc/RTCPeerConnection.d.ts

```diff
@@ -121,33 +118,19 @@ interface RTCDataChannelInit {
 }
 
 // TODO(1)
-declare enum RTCSdpType {
-  // http://dev.w3.org/2011/webrtc/editor/webrtc.html#rtcsdptype
-  'offer',
-  'pranswer',
-  'answer'
-}
+type RTCSdpType = 'offer' | 'pranswer' | 'answer' | 'rollback';
 
 interface RTCMessageEvent {
-  // http://dev.w3.org/2011/webrtc/editor/webrtc.html#event-datachannel-message
-  // At present, this can be an: ArrayBuffer, a string, or a Blob.
-  // See TODO(2)
-  data: any;
+  data: ArrayBuffer | string | Blob;
```

Comment:
According to the spec, the `data` has the type `any`. The [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent) write `Returns a DOMString, Blob or an ArrayBuffer containing the data send by the emitter`.

Not sure if it should be changed to `any` or not.